pallets/balances/src/lib.rs

```diff
@@ -798,6 +798,30 @@ where
         }
         return None;
     }
+
+    fn deposit_into_existing_identity(
```

Comment:
Should the balance be deposited into an identity or regular account? Unless we really want to move balances from accounts to identities, I'd prefer if balance is stored in accounts. cc @adamdossa

Reply:
I tend to agree. At the moment I am thinking the main reason to put POLY on an identity is to allow other keys to use it to pay for transactions. Governance, tokenomics etc. all work at the account level. In the MESH-634 story we have either an address or an identity, but this may become complicated. If we have to choose just one I think we should go with address (i.e. account).

Reply:
Let's talk about this on the standup and finalise an approach.

Reply:
@adamdossa I think this has been resolved using the new `issue` method. Please double-check.

This PR allows to deposit both to accounts and identities, and is not using the block rewards reserve for that.